docs(tutorials/blog): improve wording (#6276)
remix-run/remix: docs/tutorials/blog.md

// ...
```

> Notice we don't return a redirect if any of the input is empty (so if we have any error), we actually return the errors. These errors are available to the component via `useActionData`. It's just like `useLoaderData` but the data comes from the action after a form POST.

Reviewer: Reading the code, I think any makes more sense

> Notice we don't return a redirect if any of the input is empty (so if we have any error), we actually return the errors. These errors are available to the component via `useActionData`. It's just like `useLoaderData` but the data comes from the action after a form POST.

Reviewer 2: Looking at the code highlighting now, I think this already quite understandable tbh, as only the code about the `errors` object is highlighted

If you want to maybe change this sentence nonetheless, you could maybe doe

```suggestion
Notice we don't return a redirect if some of the input has no value set, we actually return the errors. These errors are available to the component via `useActionData`. It's just like `useLoaderData` but the data comes from the `action` after a form POST.
```
